packages/core/src/indexing/__tests__/build-indexing.test.ts

```diff
@@ -56,7 +56,7 @@ test('build for postgres connection string', async () => {
   )
   expect(indexingApi).toBeInstanceOf(PostgresIndexApi)
   await kill()
-})
+}, 10000) // TODO: CDB-XXXX creating the indexing db sometimes takes a lot of time
```

Comment:
probably should either just remove this TODO or update it with an actual ticket number before merging.